src/news/news.scss

```diff
@@ -0,0 +1,117 @@
+.news {
```

Comment:
Maybe I just don't see it, but it is better to keep styles in the same order in the codebase
http://codeguide.academy/html-css.html#css-order
you can configure liner to check it as well

Reply:
Edit